Python/testlab-to-slack/main.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START all]
+import os
+
+# [START import]
+# The Cloud Functions for Firebase SDK to set up triggers and logging.
+from firebase_functions import test_lab_fn
+
+# The requests library to send web requests to Slack.
+import requests
+# [END import]
+
+
+# [START postToSlack]
+def post_to_slack(title: str, details: str) -> any:
+    """Posts a message to Slack via a Webhook."""
+    response = requests.post(
+        url=os.environ["SLACK_WEBHOOK_URL"],
```

Comment:
Can you please use params for this? https://github.com/firebase/firebase-functions-python/pull/2/files

We should eventually have it as a Secret, but secret params aren't supported in Python yet

Reply:
hm, actually the Node sample uses `process.env` at the moment as well. Changing to params will probably change the narrative in the docs, so let's stick to env vars as you have it now